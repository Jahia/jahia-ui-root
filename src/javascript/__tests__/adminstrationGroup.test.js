import React from 'react';
import renderer from 'react-test-renderer';
import {AdministrationGroup} from '../Jahia/jahia.routes.jsx';

describe('adminstration group', () => {

    test('admin icons', () => {
        const component = renderer.create(
            <AdministrationGroup />,
        );

        let tree = component.toJSON();
        expect(tree).toMatchSnapshot();
    })
});
