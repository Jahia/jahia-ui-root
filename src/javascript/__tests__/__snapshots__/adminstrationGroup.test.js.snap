// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`administration group admin icons 1`] = `
<li
  className="_38IL5xx6MzHEv52EctnPV_"
>
  <div
    className="_3Wbxt6soOkNE807wmyu_iI"
  >
    <div
      className="_1Ai3EiPwY-pZNUwN6mrrmu"
    >
      <svg
        className=" moonstone-icon moonstone-icon_default"
        fill="none"
        size="default"
        viewBox="0 0 24 24"
      >
        <path
          clipRule="evenodd"
          d="M18.941 12c0 .306-.027.594-.064.882l1.952 1.485a.446.446 0 01.11.576l-1.85 3.114a.457.457 0 01-.564.198l-2.304-.9c-.481.351-1 .657-1.564.882l-.351 2.385a.448.448 0 01-.454.378h-3.7a.447.447 0 01-.454-.378l-.352-2.385a6.808 6.808 0 01-1.563-.882l-2.304.9a.471.471 0 01-.564-.198l-1.851-3.114a.446.446 0 01.111-.576l1.952-1.485A6.946 6.946 0 015.063 12c0-.297.027-.594.064-.882L3.175 9.633a.436.436 0 01-.111-.576l1.85-3.114a.457.457 0 01.565-.198l2.304.9a7.16 7.16 0 011.563-.882l.352-2.385A.447.447 0 0110.151 3h3.701c.232 0 .426.162.454.378l.351 2.385a6.81 6.81 0 011.564.882l2.304-.9a.471.471 0 01.564.198l1.85 3.114a.446.446 0 01-.11.576l-1.953 1.485c.037.288.065.576.065.882zM8.763 12c0 1.737 1.453 3.15 3.239 3.15 1.785 0 3.238-1.413 3.238-3.15 0-1.737-1.453-3.15-3.238-3.15-1.786 0-3.239 1.413-3.239 3.15z"
          fill="currentColor"
          fillRule="evenodd"
        />
      </svg>
    </div>
    <div
      className="_3ejht2paDjYw7X613AENsu"
    >
      <span
        className="_2UHrI1QWqxLhuY_tZNdPZo _3Jnldq9UavKB8387Kxg_LU _1Bzmq62UqnegYx7zkkgnjt _3McV6_vYIoTTSdX0ayC7p5"
      >
        primaryNavigation.administration.label
      </span>
    </div>
  </div>
</li>
`;
